uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/toil-request-ctrl.js

```diff
@@ -144,6 +160,15 @@ define([
           });
       }
 
+      /**
+       * Initialize expiryDate on UI from server date for manager's edit action
```

Comment:
I don't understand this documentation, what do you mean "for manager's edit action"